main/libcamera/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "libcamera"
+pkgver = "0.1.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dtest=true"]
+hostmakedepends = [
+    "meson",
+    "ninja",
+    "pkgconf",
+    "python-pyyaml",
+    "python-ply",
+    "python-jinja2",
+]
+makedepends = [
+    # See: https://libcamera.org/getting-started.html#dependencies
+    "libyaml-devel",
+    "libunwind-devel",
+    "udev-devel",
+    "gstreamer-devel",
+    "gst-plugins-base-devel",
+    "libevent-devel",
+    "gtest-devel",
+]
+pkgdesc = "Open source camera stack and framework"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "LGPL-2.1-or-later AND GPL-2.0-or-later"
+url = "https://libcamera.org"
+source = f"https://github.com/libcamera-org/libcamera/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "6d9b0f33939de29a2d8950924f82713981b890f9b5dc1d68a65db76a1222b15a"
+
+
+@subpackage("libcamera-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("libcamera-tools")
+def _tools(self):
+    self.pkgdesc = f"{pkgdesc} (tools)"
+    return ["usr/bin"]
```

Comment:
name this -progs and return default_progs unless it really has to be named tools for some reason